Dockerfile

```
@@ -60,48 +60,37 @@ VOLUME ["/mnt"]
 # Build and install rizin on master branch
 # Remove all build dependencies
 # Cleanup
-RUN DEBIAN_FRONTEND=noninteractive dpkg --add-architecture i386 && \
-  apt-get update && \
-  apt-get install -y \
-  curl \
-  wget \
-  gcc \
-  git \
-  bison \
-  pkg-config \
-  make \
-  glib-2.0 \
-  libc6:i386 \
-  libncurses5:i386 \
-  libstdc++6:i386 \
-  gnupg2 \
-  python-pip \
-  ${with_arm64_as:+binutils-aarch64-linux-gnu} \
-  ${with_arm32_as:+binutils-arm-linux-gnueabi} \
-  ${with_ppc_as:+binutils-powerpc64le-linux-gnu} && \
-  pip install rzpipe=="$RZ_PIPE_PY_VERSION" && \
-  cd /mnt && \
-  git clone -b "$RZ_VERSION" -q --depth 1 https://github.com/rizinorg/rizin.git && \
-  cd rizin && \
-  ./configure && \
-  make && \
-  make install && \
-  apt-get install -y xz-utils && \
-  apt-get remove --purge -y \
-  bison \
-  python-pip \
-  glib-2.0 && \
-  apt-get autoremove --purge -y && \
-  apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+# gcc git python3-pip ccache patch
+# pip3 install meson ninja
+RUN apt-get update && \
+	apt-get install -y \
+	gcc \
+	git \
+	python3-pip \
+	${with_arm64_as:+binutils-aarch64-linux-gnu} \
+	${with_arm32_as:+binutils-arm-linux-gnueabi} \
+	${with_ppc_as:+binutils-powerpc64le-linux-gnu} && \
+	pip3 install meson ninja && \
+	cd /mnt && \
+	git clone https://github.com/rizinorg/rz-pipe && \
```

set `RZ_PIPE_PY_VERSION` default to `master` and use 
```suggestion
	git clone -b "$RZ_PIPE_PY_VERSION" https://github.com/rizinorg/rz-pipe && \
```

---

Forgot to add the branch. I'm adding it.

---

Just saw the review request. Did you already add this?

---

@ret2libc search for `$RZ_VERSION`

---

No, that is different @wargio . This should use `RZ_PIPE_PY_VERSION`, not `RZ_VERSION`.

---

Sorry I thought I already added it. It's done now.